pydax/loaders/_table.py

```diff
@@ -26,8 +27,10 @@
 
 
 class CSVPandasLoader(Loader):
+    """CSV to Pandas dataframe loader.
+    """
 
-    def load(self, path: Union[_typing.PathLike, Dict[str, str]], options: SchemaDict) -> str:
+    def load(self, path: Union[_typing.PathLike, Dict[str, str]], options: SchemaDict) -> pd.DataFrame:
```

Comment:
I wonder why this wasn't caught by mypy

Reply:
yeah I was surprised too when I noticed it